### PR TITLE
feat(otel): upgrade opentelemetry to v0.75

### PR DESCRIPTION
**Pull request by leoparente**

### CODE REVIEW

File: sinker/otel/components.go [hunk 1]
```diff
@@ -53,18 +56,20 @@ func StartOtelComponents(ctx context.Context, bridgeService *bridgeservice.Sinke
...
+	// 		`set(attributes["orb-tags"], resource.attributes["orb_tags"])`,
+	// 		`set(attributes["agent-groups"], resource.attributes["agent_groups"])`,
+	// 		`set(attributes["agent-ownerID"], resource.attributes["agent_ownerID"])`,
+	// 		`set(attributes["policy-id"], resource.attributes["policy_id"])`,
+	// 		`set(attributes["policy-name"], resource.attributes["policy_name"])`,
+	// 		`set(attributes["sink-id"], resource.attributes["sink_id"])`,
+	// 		`set(attributes["format"], "otlp")`,
+	// 	}},
+	// }
+	transformSet := processor.CreateSettings{
```
Comment (lpegoraro) [inline sinker/otel/components.go:72]: why comment? if it is not going to be used anymore, clean that up.

**Review by lpegoraro — APPROVED** [review]
LGTM

File: sinker/otel/components.go (hunk 1 above)
Comment (leoparente) [reply sinker/otel/components.go:72]: oh sorry, I will fix it

Comment (leoparente) [reply sinker/otel/components.go]: done

**Review by etaques — APPROVED** [review]
LGTM

**Pull request merged**